### PR TITLE
Changes to keywords for all subcategories

### PR DESCRIPTION
**Pull request by robynk**

### CODE REVIEW

File: src/pages/personalisation/AdvocacySubcategories.js [hunk 1]
```diff
@@ -10,12 +10,16 @@ export default class AdvocacySubcategories extends BaseMultiQuestion {
...
+                .append("ombudsman complaint"),
             "Help working out a problem with a service":
-                remove("advocacy").append("mediation"),
+                remove("consumer issues discrimination disputes advocacy"),
             "Get advice on your rights":
-                remove("advocacy").append("rights advice"),
-            "Someone to speak for you": append("advocacy"),
+                remove("consumer issues discrimination disputes")
+                .remove("advocacy mediation").append("rights advice"),
+            "Someone to speak for you":
```
Comment (kane-c) [inline src/pages/personalisation/AdvocacySubcategories.js]: Remove the extra space on the end of this line

**Comment by kane-c** [comment]
Need to change line 4 to this (remove `append`):

``` js
import { remove } from "./BaseQuestion";
```

**Comment by robynk** [comment]
Need to change line 4 to this (remove append):
import { remove } from "./BaseQuestion";

Why would I remove append?  It doesn't seem obvious to me.

**Comment by kane-c** [comment]
The linter is saying that `append` is no longer used in this file, so it is superfluous. I'll remove it.

**Comment by robynk** [comment]
What about .append("rights advice"),

**Comment by kane-c** [comment]
That's with a dot - it's a separate function. Confusing!

**Comment by robynk** [comment]
Ok.  I get it.  I'll never remember it though.

**Pull request merged**